esp/scm/ecldirect.ecm

```diff
@@ -25,6 +25,8 @@ ESPStruct [nil_remove] ECLDirectException
     int LineNo;
     int Column;
     int Activity;
+    string Scope;
```

Comment:
Yes, changes to output can break some clients that do schema based validation when processing the result.  Some java based SOAP libraries are especially sensitive in this way.  

This service has barely changed since going OSS.  You'll need to add a version to the service declaration itself, something like:

`ESPservice [
    auth_feature("DEFERRED"),
    version("1.01"), 
    default_client_version("1.01")] EclDirect`